test/JDBC/expected/TestSpatialFun-vu-verify.out

```diff
@@ -0,0 +1,932 @@
+-- For STEquals
```

Comment:
Let's have comments for each section/scenario explaining what we are trying to test.

Reply:
resolved